src/sentry/utils/outcomes.py

```diff
@@ -119,6 +137,7 @@ def track_outcome(org_id, project_id, key_id, outcome, reason=None, timestamp=No
     assert isinstance(key_id, (type(None), six.integer_types))
     assert isinstance(outcome, Outcome)
     assert isinstance(timestamp, (type(None), datetime))
+    assert isinstance(event_size, (type(None), six.integer_types))
```

Comment:
Maybe unrelated with this PR, but looking at the size method on the Event class, it is the size in characters of the json serialization of the data field of the event. That may be different than the size in bytes and it is the size of an event post normalization and can be tweaked during event save. 
IS this the expected way to measure event size ?

Reply:
I think there is an open question about what constitutes an event size. The goal of this pull request is to add this metric to the outcome, knowing that the implementation of event size for billing purposes is still to come.